docs/installation/server-config/security.rst

```diff
@@ -1,4 +1,172 @@
+        producer.confluent.monitoring.interceptor.ssl.keystore.location=/etc/kafka/secrets/kafka.client.keystore.jks
+        producer.confluent.monitoring.interceptor.ssl.keystore.password=confluent
+        producer.confluent.monitoring.interceptor.ssl.key.password=confluent
+        producer.confluent.monitoring.interceptor.security.protocol=SASL_SSL
+        producer.confluent.monitoring.interceptor.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="client" password="client-secret";
+        producer.confluent.monitoring.interceptor.sasl.mechanism=PLAIN
+
+        # Consumer Confluent Monitoring Interceptors for Control Center streams monitoring
+        consumer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
+        consumer.confluent.monitoring.interceptor.ssl.truststore.location=/etc/kafka/secrets/kafka.client.truststore.jks
+        consumer.confluent.monitoring.interceptor.ssl.truststore.password=confluent
+        consumer.confluent.monitoring.interceptor.ssl.keystore.location=/etc/kafka/secrets/kafka.client.keystore.jks
+        consumer.confluent.monitoring.interceptor.ssl.keystore.password=confluent
+        consumer.confluent.monitoring.interceptor.ssl.key.password=confluent
+        consumer.confluent.monitoring.interceptor.security.protocol=SASL_SSL
+        consumer.confluent.monitoring.interceptor.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="client" password="client-secret";
+        consumer.confluent.monitoring.interceptor.sasl.mechanism=PLAIN
+
+    The exact settings you need depend on what SASL mechanism your Kafka cluster is using and how your SSL certificates
+    are signed. For full details, see the `Security section of the Kafka documentation <http://kafka.apache.org/documentation.html#security>`__.
+
+#.  Start KSQL server with your configuration file specified.
+
+    .. code:: bash
+
+        $ <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
+
+    .. tip:: After KSQL is started, you can view your settings with the ``SHOW PROPERTIES;`` KSQL command.
+
+---------------------------------------------------
+Using KSQL with a secured Confluent Schema Registry
+---------------------------------------------------
+
+If you are using KSQL with HTTPS to Confluent Schema Registry, you must set the ``ksql.schema.registry.url`` to HTTPS
+(as shown in the example configuration) and set the ``KSQL_OPTS`` environment variable to define the credentials to use
+when communicating with the Confluent Schema Registry. This must be run before starting the KSQL server.
+
+.. code:: bash
+
+    # Define KSQL security credentials when communicating with the Confluent Schema Registry via HTTPS
+    $ export KSQL_OPTS="-Djavax.net.ssl.trustStore=/etc/kafka/secrets/kafka.client.truststore.jks
```

Comment:
Do we want to hard code this to '/etc/kafka/secrets/...'?

We also may want to make it clear that the user will need to create this thrust store, and point them to the Kafka docs for the instructions to do this.

http://kafka.apache.org/documentation.html#security_ssl

Reply:
re: whether or not this is recommend procedure - i don't know. 

i have added tip about creating truststore to doc revision.